docs/submission/validating-molecular-data.md

```diff
@@ -266,37 +267,37 @@ Sequencing data of both `BAM` and `FASTQ` type files are accepted, with the exce
+|  **submitter_read**\_ **group_id**  | read_groups |  ![Required](/assets/submission/dictionary-required.svg)  |  The unique identifier of a read group; must be unique within each payload. |   String values that meet the regular expression `^[a-zA-Z0-9\\-_:\\.']+$` or `null`. |  
+|  **platform_unit**    | read_groups |  ![Required](/assets/submission/dictionary-required.svg)  |  Unique identifier including the {FLOWCELL_BARCODE}.{LANE}.{SAMPLE_BARCODE}. The {FLOWCELL_BARCODE} refers to the unique identifier for a particular flow cell. The {LANE} indicates the lane of the flow cell and the {SAMPLE_BARCODE} is a sample/library-specific identifier. For non-multiplex sequencing, platform unit and read group have a one-to-one relationship.  |     Any string value.                 |  
+|  **is_paired_end**  | read_groups |  ![Required](/assets/submission/dictionary-required.svg)  |  Indicate if paired-end sequencing was performed.                                                           |  true, false         |  
+|  **file_r1**  | read_groups |  ![Required](/assets/submission/dictionary-required.svg)  |  Name of the sequencing file containing reads from the first end of a sequencing run. | Any string value. Must match a fileName identified in the `files` section.|  
+|  **file_r2** | read_groups |  ![Required](/assets/submission/dictionary-required.svg)  |  Name of the sequencing file containing reads from the second end of a paired-end sequencing run. Required if and only if paired-end sequencing was done.|  Any string value or `null`. Must match a fileName identified in the `files` section.    |  
+|  **read_length_r1**  | read_groups |  ![Required](/assets/submission/dictionary-required.svg) |  Length of sequencing reads in `file_r1`; this corresponds to the number of sequencing cycles of the first end.                                                                                                                    |       Integer with a minimum value of 20  or `null`.            |  
+|  **read_length_r2**  | read_groups |  ![Required](/assets/submission/dictionary-required.svg)  |  Length of sequencing reads in `file_r2`; this corresponds to the number of sequencing cycles of the second end. |  Integer with a minimum value of 20  or `null`. |  
+|  **insert_size**  | read_groups |  ![Required](/assets/submission/dictionary-required.svg)  |  For paired-end sequencing, the average size of sequences between two sequencing ends. Required only for paired-end sequencing. |    Integer with a minimum value of 0  or `null`. |  
+|  **sample_barcode**| read_groups |![Required](/assets/submission/dictionary-required.svg)                                              |  According to the SAM specification, this is the expected barcode bases as read by the sequencing machine in the absence of errors.      |    Any string value or `null`.                  |  
```

Comment:
should this say BAM instead of SAM?

Reply:
no this is correct; its the SAM specification SAMS/BAMS/CRAMS are all related file types.